src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -29,6 +32,8 @@ public CCJContent(Claim claim) {
         this.amountToPayByDefendant = formatMoney(
             ((AmountBreakDown) claim.getClaimData().getAmount()).getTotalAmount()
         );
+        this.claimantName = claim.getClaimData().getClaimant().getName();
```

Comment:
I would have expected PDF changes here to add the signerName and signerRole...

Reply:
they are already there, please check template

Reply:
please point me to where I could find these variables, I must be blind but I can't see them:
https://github.com/hmcts/cmc-claim-store/blob/c34d752ec7472c6e9455d423afa9fff7585beff9/src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

Reply:
signature name and signature role they are not part of this pdf as per
https://tools.hmcts.net/jira/browse/ROC-2057

**Ben's Comment:** 

**New content for PDF needed. After the fields we currently display in the PDF we need:
Declaration
((Claimant name)) [in future the person signing the declaration could be someone other than the claimant, but for now it's only this option]
I declare that the details I have given are true to the best of my knowledge.
[date of request of CCJ]**